client/src/shared/Footer/Footer.tsx

```
@@ -48,9 +48,10 @@ const Footer = () => {
             </a>
           ))}
         </div>
+        <hr></hr>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)